packages/container/lib/container.js

```diff
@@ -23,9 +20,6 @@ function Container(registry, options) {
   this._registry = registry || (function() {
     Ember.deprecate("A container should only be created for an already instantiated registry. For backward compatibility, an isolated registry will be instantiated just for this container.");
 
-    // TODO - See note above about transpiler import workaround.
-    if (!Registry) { Registry = requireModule('container/registry')['default']; }
```

Comment:
Due to the requirement to be ES3 safe we do not get many of the features that allow cycles / late bindings.  Will have to revert this particular change.